src/services/users.js

```diff
@@ -73,9 +78,11 @@ const signInUserByEmail = async (email, password) => {
 	}
 
 	const body = { id: user.id, email: user.email }
-	const token = jwt.sign({ user: body }, config("TOKEN_SECRET"))
+	const accessToken = jwt.sign({ user: body }, config("TOKEN_SECRET"),
+		{expiresIn: '900s'})
+	const refreshToken = crypto.randomBytes()
```

Comment:
not sure this works. the function was put there so its usage is looked into to know how to actually call it to get the string

Reply:
the generated refresh token here should be stored as an update for the user